platform/security/filter/security-filter-web-sso/src/test/java/org/codice/ddf/security/filter/websso/WebSSOFilterTest.java

```diff
@@ -53,8 +53,8 @@
 
     @Test
     public void testInit() throws ServletException {
-        Logger logger = (Logger) LoggerFactory.getLogger(WebSSOFilter.class);
-        logger.setLevel(Level.DEBUG);
+        Logger LOGGER = (Logger) LoggerFactory.getLogger(WebSSOFilter.class);
```

Comment:
Only use all-caps for variable names when they are private static final. In this case, it should be "logger".